source/Mlos.Notebooks/SmartCacheOptimization.ipynb

```diff
@@ -103,8 +103,8 @@
       "text/plain": [
        "smart_cache_config\n",
        "implementation: [LRU, MRU]\n",
-       "lru_cache_config.cache_size: {1, 1 + 1, ... , 4096}\n",
-       "mru_cache_config.cache_size: {1, 1 + 1, ... , 4096}"
+       "lru_cache_config.cache_size: {1, 2, 3, ... , 4096}\n",
```

Comment:
this is a meaningful change in output because of updates in the code. The remaining changes are incidental. yay!